Doc/library/heapq.rst

```diff
@@ -82,6 +84,59 @@ The following functions are provided:
    on the heap.
 
 
+For max-heaps, the reverse of a heap, the following functions are provided:
+
+
+.. function:: heapify_max(x)
+
+   Transform list *x* into a max-heap, in-place, in linear time.
+
+   .. versionadded:: next
+
+
+.. function:: heappush_max(heap, item)
+
+   Push the value *item* onto the max-*heap*, maintaining the heap invariant.
```

Comment:
I would prefer "`max-heap *heap*`". *heap* is a max-heap, and it's better to repeat it. Within the same paragraph, we can just refer to *heap*, but the first occurrence should be exact.